reversion/models.py

```diff
@@ -1,352 +1,34 @@
-from collections import defaultdict
-from itertools import chain, groupby
+import import_string
 
-from django.apps import apps
 from django.conf import settings
-from django.contrib.contenttypes.fields import GenericForeignKey
-from django.contrib.contenttypes.models import ContentType
-from django.core import serializers
-from django.core.exceptions import ObjectDoesNotExist
-from django.core.serializers.base import DeserializationError
-from django.db import IntegrityError, connections, models, router, transaction
-from django.db.models.deletion import Collector
-from django.db.models.functions import Cast
-from django.utils.encoding import force_str
-from django.utils.functional import cached_property
-from django.utils.translation import ugettext
-from django.utils.translation import gettext_lazy as _
 
-from reversion.errors import RevertError
-from reversion.revisions import (_follow_relations_recursive,
-                                 _get_content_type, _get_options)
 
+BACKENDS = [
+    app_name.split('.')[-1] for app_name in settings.INSTALLED_APPS
+    if isinstance(app_name, str) and app_name.startswith('reversion.backends.')
+]
```

Comment:
What's point of having multiple backends enabled?

Reply:
wouldn't an explicit definition be better?